src/boot/efi/stub.c

```diff
@@ -357,14 +356,69 @@ static void install_addon_devicetrees(
         }
 }
 
+static inline void iovec_array_append(struct iovec **arr, size_t *n_arr, struct iovec elem) {
+        *arr = xrealloc(*arr, *n_arr * sizeof(struct iovec), (*n_arr + 1)  * sizeof(struct iovec));
+        (*arr)[(*n_arr)++] = elem;
+}
+
+static void measure_ucode_addons_and_create_final_list(
+                struct iovec initrds[static _INITRD_MAX],
+                struct iovec **all_initrds,
+                size_t *n_all_initrds,
+                NamedAddon *ucode_addons,
+                size_t n_ucode_addons,
+                int *sections_measured) {
+        EFI_STATUS err;
```

Comment:
We always add a space after a multiline function declaration before we begin variable declarations